instructor/ui/sketch_tool.js

```diff
@@ -129,7 +131,8 @@ function draw_line(x1 ,y1 ,x2 ,y2, color){
 
 }
 
-function nameXAxis(condition){
+function nameXAxis(condition, this_paper){
+    paper = this_paper;
```

Comment:
Ah, I see. Can you move that step and `paper.view.update()` into the calling function? `nameYAxis` doesn't do this setup so it would be good to be consistent